packages/core/src/protocol-helpers.ts

```diff
@@ -20,7 +20,7 @@ import { ReadableStream } from 'web-streams-polyfill/ponyfill/es2018';
 export default class ProtocolHelpers {
 
   // set contentType (extend with more?)
-  public static updatePropertyFormWithTemplate(form: TD.Form, tdTemplate: WoT.ThingDescription, propertyName: string) {
+  public static updatePropertyFormWithTemplate(form: TD.Form, tdTemplate: WoT.ExposedThingInit, propertyName: string) {
```

Comment:
shouldn't be `tdTemplate?: WoT.ExposedThingInit` I'am wondering why is working right now. Cause other parts of the code base, we are passing something that might be undefined...